contrib/vagrant/build.sh

```diff
@@ -2,6 +2,7 @@
 
 cd /home/vagrant/enclave
 
+export CLANG=/usr/bin/clang-12
```

Comment:
Is there a reason why CLANG isn't set by default? Package manager installation doesn't set the ENV?

Reply:
Exactly. Installing the newest clang-12 from their apt repository doesn't set the env, doesn't use update-alternatives, doesn't make any symlinks. So I decided that solving it with an environment variable will take the least effort.